js/MVP/hooks/moteplanlegger.js

```diff
@@ -4,5 +4,7 @@ import { getMoteadmin } from '../services/moteadmin';
 const MOTEPLANLEGGER = 'moteplanlegger';
 
 export const useMoteplanlegger = () => {
-  return useQuery(MOTEPLANLEGGER, getMoteadmin);
+  return useQuery(MOTEPLANLEGGER, getMoteadmin, {
+    retry: 0,
```

Comment:
hvorfor retry 0? større fare for feil på nettverksbrudd

Reply:
Endepunktet returnerer `4xx` ved ingen møter som ikke er uvanlig og med default retries ender hooken opp med å ta ish 3 sekunder hver gang vi gjør kallet som blant annet er på landingssiden. Kanskje andre options er bedre? Én retry?

Reply:
ahh. 404 er behandlet som error med fetch, men ikke med axios

Reply:
Beste blir nok da å endre til axios. Om ikke, kan du nok legge til i fetch-wrapperen en (if 404) { returner tom respons }) eller noe

Reply:
Ser ut som kanskje axios også behandler 40x som error, men det er customizable: https://axios-http.com/docs/handling_errors

Reply:
Fikser på wrapperen og fjerner 0 retries så kan vi opdatere til Axios i en annen PR

Reply:
cool